src/Microsoft.Identity.Client/TokenCache.cs

```diff
@@ -37,14 +37,75 @@
 
 namespace Microsoft.Identity.Client
 {
+    internal class TelemetryTokenCacheAccessor : TokenCacheAccessor
+    {
+        // The content of this class has to be placed outside of its base class TokenCacheAccessor,
+        // otherwise we would have to modify multiple implementations of TokenCacheAccessor on different platforms.
+        public void SaveAccessToken(string cacheKey, string item, RequestContext requestContext)
```

Comment:
But they are not real public. They are methods of an internal class. And they need to be called by other internal classes outside of their inheritance family, so they need to be public.

Their pre-existing [base class also defined them as public](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/dev/src/Microsoft.Identity.Client/Features/TokenCache/TokenCacheAccessor.cs#L43), BTW.